jwst/assign_wcs/util.py

```diff
@@ -473,8 +477,8 @@ def get_object_info(catalog_name=None):
                                  xcentroid=row['xcentroid'],
                                  ycentroid=row['ycentroid'],
                                  sky_centroid=row['sky_centroid'],
-                                 abmag=row['abmag'],
-                                 abmag_error=row['abmag_error'],
+                                 abmag=row['isophotal_abmag'],
```

Comment:
Ah good, so all we have to do is change the assignments here and then not worry about having to change the places where `abmag` and `abmag_error` are used in other parts of the code. 👍